packages/experimental-nextjs-app-support/src/ssr/index.ts

```diff
@@ -1,17 +1,138 @@
-  useReadQuery,
-  useSuspenseQuery,
+import {
+  useBackgroundQuery as _useBackgroundQuery,
+  useFragment as _useFragment,
+  useQuery as _useQuery,
+  useReadQuery as _useReadQuery,
+  useSuspenseQuery as _useSuspenseQuery,
 } from "@apollo/client/index.js";
+
+/**
+ * @deprecated
+ * This import has been renamed to `InMemoryCache` and moved to `"@apollo/experimental-nextjs-app-support"`.
+ * Please update your import to
+ * ```ts
+ * import { InMemoryCache } from "@apollo/experimental-nextjs-app-support";
+ * ```
+ */
+export const NextSSRInMemoryCache = InMemoryCache;
+/**
+ * @deprecated
+ * This import has been renamed to `ApolloClient` and moved to `"@apollo/experimental-nextjs-app-support"`.
+ * Please update your import to
+ * ```ts
+ * import { ApolloClient } from "@apollo/experimental-nextjs-app-support";
+ * ```
+ */
+export const NextSSRApolloClient = ApolloClient;
+/**
+ * @deprecated
+ * This import has been renamed to `resetApolloClientSingletons` and moved to `"@apollo/experimental-nextjs-app-support"`.
+ * Please update your import to
+ * ```ts
+ * import { resetApolloClientSingletons } from "@apollo/experimental-nextjs-app-support";
+ * ```
+ */
+export const resetNextSSRApolloSingletons = resetApolloClientSingletons;
+/**
+ * @deprecated
+ * This import has moved to `"@apollo/experimental-nextjs-app-support"`.
+ * Please update your import to
+ * ```ts
+ * import { SSRMultipartLink } from "@apollo/experimental-nextjs-app-support";
+ * ```
+ */
+export const SSRMultipartLink = _SSRMultipartLink;
+/**
+ * @deprecated
+ * This import has moved to `"@apollo/experimental-nextjs-app-support"`.
+ * Please update your import to
+ * ```ts
+ * import { DebounceMultipartResponsesLink } from "@apollo/experimental-nextjs-app-support";
+ * ```
+ */
+export const DebounceMultipartResponsesLink = _DebounceMultipartResponsesLink;
+/**
+ * @deprecated
+ * This import has moved to `"@apollo/experimental-nextjs-app-support"`.
+ * Please update your import to
+ * ```ts
+ * import { RemoveMultipartDirectivesLink } from "@apollo/experimental-nextjs-app-support";
+ * ```
+ */
+export const RemoveMultipartDirectivesLink = _RemoveMultipartDirectivesLink;
+/**
+ * @deprecated
+ * This import has moved to `"@apollo/experimental-nextjs-app-support"`.
+ * Please update your import to
+ * ```ts
+ * import { ApolloNextAppProvider } from "@apollo/experimental-nextjs-app-support";
+ * ```
+ */
+export const ApolloNextAppProvider = _ApolloNextAppProvider;
+/**
+ * @deprecated
+ * This import has moved to `"@apollo/experimental-nextjs-app-support"`.
+ * Please update your import to
+ * ```ts
+ * import type { TransportedQueryRef } from "@apollo/experimental-nextjs-app-support";
+ * ```
+ */
+export type TransportedQueryRef<
+  TData = unknown,
+  TVariables = unknown,
+> = _TransportedQueryRef<TData, TVariables>;
+/**
+ * @deprecated
+ * Importing this hook from this package is not necessary anymore - you can directly import it from `@apollo/client`.
```

Comment:
```suggestion
 * Importing `useBackgroundQuery` from this package is no longer necessary. Import it directly from `@apollo/client` instead.
```

[nit] I'd break this up into two sentences as I think it reads a bit better. Feel free to keep what you have though if you don't like this change, especially since this is just what you see in the editor. If you do like this, the other hook deprecations could also need this change to keep this consistent.

Reply:
Thank you!